### PR TITLE
feat(popup): popup creation

### PR DESCRIPTION
**Pull request by AlexandreBarbier**

### CODE REVIEW

File: packages/doc/src/stories/PopUp.stories.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  title: 'Components/PopUp',
+  component: MkrPopUp,
+  argTypes: {
+
+  },
+};
+
+const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  template: '<mkr-pop-up v-bind="$props"><template v-slot:anchor><mkr-contained-button>button</mkr-contained-button></template><mkr-card :elevated="true"><mkr-contained-button>test</mkr-contained-button></mkr-card></mkr-pop-up>',
```
Comment (GuillaumeBAECHLER) [inline packages/doc/src/stories/PopUp.stories.js:13]: We can use backtick and indent for readability

File: packages/mikado_reborn/src/components/PopUp/PopUp.vue [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    const anchor = this.$refs.anchor as HTMLElement;
+    const content = this.$refs.content as HTMLElement;
+
+    this.popperInstance = createPopper(anchor.children[0], content.children[0] as HTMLElement, {
+      placement: this.placement,
+      modifiers: [
+        {
+          name: 'offset',
+          options: {
+            offset: [0, 4],
```
Comment (GuillaumeBAECHLER) [inline packages/mikado_reborn/src/components/PopUp/PopUp.vue:37]:
```suggestion
            offset: [0, 8],
```

File: packages/mikado_reborn/src/components/PopUp/PopUp.vue [hunk 3]
```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="mkr__popup">
+    <div ref="anchor" @click="handleButtonClick">
+      <slot name="anchor"/>
+    </div>
+    <div ref="content" :class="[{'mkr__popup--hidden': !isTooltipVisible}]">
```
Comment (GuillaumeBAECHLER) [inline packages/mikado_reborn/src/components/PopUp/PopUp.vue:6]:
```suggestion
    <div ref="content" :class="{'mkr__popup--hidden': !isTooltipVisible}">
```

**Review by GuillaumeBAECHLER — CHANGES_REQUESTED** [review]

File: packages/doc/src/stories/PopUp.stories.js (hunk 1 above)
Comment (AlexandreBarbier) [reply packages/doc/src/stories/PopUp.stories.js:13]: 👍

**Review by GuillaumeBAECHLER — APPROVED** [review]

**Review by hhochart29 — APPROVED** [review]
Top 👌  !

**Pull request merged**